packages/app-project/src/screens/ClassifyPage/components/YourProjectStats/useYourProjectStats.js

```diff
@@ -96,10 +99,22 @@ async function fetchStats({ endpoint, projectID, userID, token }) {
   return null
 }
 
+/**
+ * Optimistically increment project stats, without revalidating the SWR cache.
+ * @param {string} projectID 
+ * @param {string} userID 
+ * @returns the mutated stats data
+ */
+export function updateYourStats(projectID, userID) {
+  return incrementStats(mutate, projectID, userID)
+}
+
 export default function useYourProjectStats({ projectID, userID }) {
   const token = usePanoptesAuthToken()
 
   // only fetch stats when a userID is available. Don't fetch if no user signed in.
   const key = token && userID ? { endpoint, projectID, userID, token } : null
-  return useSWR(key, fetchStats, SWROptions)
+  const fetchedStats = useSWR(key, fetchStats, SWROptions)
+  mutate = fetchedStats.mutate
```

Comment:
Does anyone know what `mutate` evaluates to here when the key is null? The old stats code kept a running count of classifications in the current tab, if you weren’t logged in. Just so you’d have some feedback that your classification had been saved, after you pressed Done.